### PR TITLE
Update python requirement to add boto3

### PR DESCRIPTION
**Pull request by farhaanbukhsh**
This PR updates and adds boto3 to support new way of interacting with S3 bucket.

### CODE REVIEW

**Review by shimulch — APPROVED** [review]
LGTM :+1:

- [x] I read through the code

File: requirements/production.in [hunk 1]
```diff
@@ -11,3 +11,5 @@ mysqlclient
 newrelic
 python-memcached
 PyYAML
+boto
```
Comment (0x29a) [inline requirements/production.in:14]: Nit: do we need this here?

**Review by 0x29a — APPROVED** [review]
:+1:

- [x] I read through the code.

File: requirements/production.in (hunk 1 above)
Comment (farhaanbukhsh) [reply requirements/production.in:14]: just to be sure

**Pull request merged**